app/src/main/java/io/punchtime/punchtime/ui/activities/MainActivity.java

```diff
@@ -255,7 +255,7 @@ public void onConfigurationChanged(Configuration newConfig) {
 
     @Override
     public void onFirebaseLoggedIn(final AuthData authData) {
-        setFragment(new DashboardFragment());
+        if(!isLaunchedFromIntent) setFragment(new DashboardFragment());
```

Comment:
this is the line that makes the app always launch in Dashboard mode instead of remembering what fragment the app quitted in I think